include/bluetooth/mesh/scheduler_srv.rst

```diff
@@ -0,0 +1,222 @@
+.. _bt_mesh_scheduler_srv_readme:
+
+Scheduler Server
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Scheduler Server model holds the current state of the Scheduler Register, and performs scheduling of the configured actions.
+The model notifies appropriate :ref:`bt_mesh_scene_srv_readme` and :ref:`bt_mesh_onoff_srv_readme` instances about actions in progress.
+
+The Scheduler Server relies on the :ref:`bt_mesh_time_srv_readme` that is provided during initialization.
+However, it is the application that should handle the Time Server events and notify the Scheduler Server if the UTC time or the Time Zone have changed.
+
+The Scheduler Server model adds two model instances in the composition data:
+
+* Scheduler Server
+* Scheduler Setup Server
+
+The two model instances share the states of the Scheduler Server, but accept different messages.
+This allows for a fine-grained control of the access rights for the Scheduler Server states, as the two model instances can be bound to different application keys.
+
+* Scheduler Server manages the scheduled actions.
+* Scheduler Setup Server provides functionality for configuration and recalling of actions in the Scheduler Server's Scheduler Register state.
+
+Operation
+=========
+
+The Scheduler models perform conversion of the configuration parameters from incoming client messages into :ref:`international atomic time (TAI) <bt_mesh_time_tai_readme>`.
+The configuration parameters with calculated time closest to the current time are scheduled as actions.
+If an action requires rescheduling when the scheduled time has expired, the Scheduler Server calculates new time and repeats the scheduling procedure.
+However, the Scheduler Server skips configuration parameters not allowing to calculate the exact time of the action.
+Such actions will never be fired.
```

Comment:
Fired or executed?

Reply:
I think `executed` the more correct here. I'll fix it.

Reply:
Maybe fix it in the next paragraph too :) These are the only two occurrences that I've found.